src/cmds/tail.c

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file
+ * @brief head-util
+ *
+ * @date 07.09.18
+ * @author Filipp Chubukov
+ */
+
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <ctype.h>
+
+#define SIZE 100
+
+static void read_last_n_lines(FILE *fp, int n){
+	  int lines_checked = 0;  
+
+    unsigned long long position_of_last_line_to_read; 
+    char buf_str[2*SIZE]; 
+  
+    if (fseek(fp, 0, SEEK_END)) {
+        printf("fseek() failed"); 
+    	return ;	
+    }
+    else
```

Comment:
Why is else here? 'if' block has 'return' it's enough for me.

Reply:
Oh yeah, will change it right now.